magento/magento2
# Fix ui-select with long item list making page longer

### Description (*)

Ui-Select elements with really long options lists also made the page longer when dropdown is open, even though they were only displayed as a smaller dropdown (max-height 46rem), and would allow scrolling past the page footer.

### Related Pull Requests


### Fixed Issues (if relevant)


### Manual testing scenarios (*)

1. Create ui-select component with many options (100 - 200 +) on a shorter page.
2. Page should remain same length even with long options dropdown open.

### Questions or comments


### Contribution checklist (*)
